src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTask.java

```diff
@@ -49,11 +50,10 @@ public void deleteMessagesInEnvelopesDlq() throws ServiceBusException, Interrupt
             while (message != null) {
                 if (canBeDeleted(message)) {
                     logMessage(message);
-                    messageReceiver.complete(message.getLockToken());
+                    messageReceiver.completeAsync(message.getLockToken());
```

Comment:
Why async? I think it's best to leave `complete`, have all the exceptions thrown if something goes wrong and not start a new message until the previous one has been dealt with.

Reply:
@lgonczar Changed it back to `complete`